react-homework/src/App.jsx

```diff
@@ -0,0 +1,60 @@
+mport React from 'react';
+import './App.css';
+
+import Cars from './components/Cars';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+      this.state = { 
```

Comment:
Nu e gresit, insa cel mai ok e ca in constructor sa initializezi state-ul cu valori default, apoi aceste valori sa le setezi pe stare initial in ComponentDidMount. (se presupune ca astfel de date ar trebui sa vina de pe un server, call ce ar fi existat in componentDidMount).

Locul de a schimba starea intr-un mod specific, e in ComponentDidMount, ComponentDidUpdate (de aceasta metoda vom discuta saptamana aceasta)

Eu as fi initializat astfel: this.state = { laptops: [] }

Fiind tema 2, stiu ca nu a fost prezentata inca metoda componentDidMount() -> e ok codul pentru tema 2 :)